<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bank</class>
 <widget class="QMainWindow" name="bank">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>bank</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: rgb(255, 255, 255)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>190</y>
      <width>301</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>signIn</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_passowrd">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: rgb(176, 255, 216);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(45, 46, 40);
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}</string>
       </property>
       <property name="maxLength">
        <number>32768</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>username</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>passowrd</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_username">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: rgb(176, 255, 216);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(45, 46, 40);
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
     background-color:rgb(138, 146, 255) border: 2px ;
	border-radius: 2px;
	padding: 2px;
}</string>
       </property>
       <property name="text">
        <string>login</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_stat">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>450</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>[+]status</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pic">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>311</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>361</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>signup</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_passsword2">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: rgb(176, 255, 216);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(45, 46, 40);
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_clid2">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: rgb(176, 255, 216);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(45, 46, 40);
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_username2">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: rgb(176, 255, 216);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(45, 46, 40);
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_signup">
       <property name="styleSheet">
        <string notr="true">QPushButton {
     background-color:rgb(138, 146, 255) border: 2px ;
	border-radius: 2px;
	padding: 2px;
}</string>
       </property>
       <property name="text">
        <string>Signup</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1040</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionRegister"/>
   </widget>
   <addaction name="menufile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRegister">
   <property name="text">
    <string>Register</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
